lib/mumukit/server/test_pipeline.rb

```diff
@@ -12,15 +12,18 @@ def evaluate!
   end
 
   def generate_feedback!
-    @feedback = @server.run_feedback! @request,
+    feedback = @server.run_feedback! @request,
                                      struct(test_results: @test_results, expectation_results: @expectation_results)
+    @inline_errors, @feedback = feedback.partition {|it| it.key? :type}
```

Comment:
Please check if feedback is a string in order to keep compatibility like in https://github.com/mumuki/mumukit/pull/67#issuecomment-352778410